doc/rel_notes/0.7.0.txt

```diff
@@ -98,6 +98,8 @@ OpenCV Plugin
  * Added unit tests to verify that the lens distortion functions in MAP-Tk
    produce equal results to those routines in OpenCV.
 
+ * Fixed error where the size of OCV image memory was incorrectly calculated.
+
```

Comment:
Updated change as per request.

On Mon, Nov 23, 2015 at 2:28 PM, Matt Leotta <notifications@github.com>
wrote:

> In doc/rel_notes/0.7.0.txt
> <https://github.com/Kitware/maptk/pull/102#discussion_r45648525>:
>
> > @@ -98,6 +98,8 @@ OpenCV Plugin
> >   * Added unit tests to verify that the lens distortion functions in MAP-Tk
> >     produce equal results to those routines in OpenCV.
> >
> > + * Fixed error where the size of OCV image memory was incorrectly calculated.
> > +
>
> Can you move this comment to the "Fixes since v0.6.0" section at the end
> of this document and create a new "OpenCV Plugin" sub-section there?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/maptk/pull/102/files#r45648525>.
>



-- 
*Linus Sherrill - *R&D Engineer
Kitware, Inc.
28 Corporate Drive
Clifton Park, NY 12065-8662
E: linus.sherrill@kitware.com
P: 518.881.4400